### PR TITLE
x11-libs/vte version bump to 0.28.2-r208

### PR DESCRIPTION
**Pull request by djarosz**
@gentoo/gnome
# Changes
	- version bump
	- included patch for vte2 fixing bug described below
	- added call to epatch_user

# Bug details
If you are using gnome-terminal or xfce4-terminal, you may have hit a bug in vte3.
A fix is around since at least 2010, which found its way into some
(but not all) distribution packages, and even into upstream (3 years later,
but unchanged).

## Bugzilla and other links
- Original (?):
	https://bugzilla.gnome.org/show_bug.cgi?id=542087
- One of its duplicate:
	https://bugzilla.gnome.org/show_bug.cgi?id=686097
- reported mosh bug related to this problem:
	https://github.com/mobile-shell/mosh/issues/660
- vte upstream commit:
	https://git.gnome.org/browse/vte/commit/?id=88e8e89560a62d0981ce2b18974a230d0a07dbdd
- used backported patch for vte2:
	https://github.com/pld-linux/vte0/commit/1e8dce16b239e5d378b02e4d04a60e823df36257

### CODE REVIEW

File: x11-libs/vte/vte-0.28.2-r208.ebuild [hunk 1]
```diff
@@ -0,0 +1,128 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI="5"
```
Comment (SoapGentoo) [inline x11-libs/vte/vte-0.28.2-r208.ebuild]: please modernize this ebuild to `EAPI=6`

**Comment by monsieurp** [comment]
Thanks for the PR. Could also file a bug in Bugzilla as well and assign it to the Gnome team? Cheers.

**Comment by monsieurp** [comment]
Please do not merge master back into the PR. This is not OK and cocks up the whole process. Could you file another PR with the same changes? Thanks.

**Comment by djarosz** [comment]
Sorry. New PR is here: https://github.com/gentoo/gentoo/pull/2444

**Pull request closed without merging**